packages/client/src/generation/utils/buildDMMF.ts

```diff
@@ -1,18 +1,24 @@
+import { DMMF } from '@prisma/generator-helper'
 import lzString from 'lz-string'
 
+import { escapeJson } from '../TSClient/helpers'
+
 /**
  * Creates the necessary declarations to embed the generated DMMF into the
  * generated client. It compresses the DMMF for the data proxy engine.
```

Comment:
```suggestion
 * generated client. It compresses the DMMF for the data proxy engine. For
 * data proxy, the full DMMF is exported, otherwise `schema` is removed.
```